nirc_ehr/resources/queries/study/necropsy.js

```diff
@@ -1,10 +1,48 @@
 require("ehr/triggers").initScript(this);
 var triggerHelper = new org.labkey.nirc_ehr.query.NIRC_EHRTriggerHelper(LABKEY.Security.currentUser.id, LABKEY.Security.currentContainer.id);
 
+var deathIdMap = {};
+
+function onInit(event, helper){
+
+    LABKEY.Query.selectRows({
```

Comment:
I think we also need to look at deathsInTransaction. For the case when a user is filling in the death info and necropsy info at the same time.